test/bulk.test.js

```diff
@@ -102,6 +105,110 @@ describe("bulk", function() {
   /**
    *
    */
```

Comment:
here are the tests. Basically it takes 1 record from the bulk insert above and inserts some attachments on it. It copies over a requestTemplate.txt file into the directory to be zipped, replaces a regex with a valid Id, zips it up, and sends the request off as a stream of the zip